### PR TITLE
Document installation from source git repository

### LINKED ISSUE #1018: Document how to install development version of foreman-ansible-modules
##### SUMMARY
I try to install the development-version of the foreman-ansible-modules following [standard procedure](https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-a-collection-from-a-git-repository).  Installation fails, see below.

Please document how to install the development version of the foreman-ansible-modules.

##### ISSUE TYPE
 - Documentation

##### ANSIBLE VERSION

```
ansible 2.9.10
  python version = 3.8.5 (default, Sep  5 2020, 10:50:12) [GCC 10.2.0]```
```

##### STEPS TO REPRODUCE

Have a requirements.yml like this:
```
---
collections:
  - name: https://github.com/theforeman/foreman-ansible-modules.git
    type: git
    version: develop
```

Run the install command:
```
$ ansible-galaxy collection install --ignore-errors -r collections/requirements.yml
Process install dependency map
ERROR! Collection artifact at '/home/marco/.ansible/tmp/ansible-local-126360f6gj6yxh/tmpu98ri6ri/foreman-ansible-modulesrdoar3kk.git' is not a valid tar file.
```

As an alternative, run:
```
$ ansible-galaxy collection install git+https://github.com/theforeman/foreman-ansible-modules.git,develop
Process install dependency map
ERROR! Invalid collection name 'git+https', name must be in the format <namespace>.<collection>. Please make sure namespace and collection name contains characters from [a-zA-Z0-9_] only.
```
This fails because of the dashes in the repositories name.

##### EXPECTED RESULTS

I would expect the installation to succeed following the standard installation procedure.

##### ACTUAL RESULTS

Installation fails.  See above.

### PR DESCRIPTION
**Pull request by wbclark**
Closes #1018

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,46 @@ The collection is also available as `ansible-collection-theforeman-foreman` from
...
+  - name: https://github.com/theforeman/foreman-ansible-modules.git
+    type: git
+    version: devel
+```
+
+And install all specified requirements with `ansible-galaxy install -r requirements.yml`
+
+### Building and Installing the Collection Locally
+
+For older versions of Ansible (< 2.10) where the above approach is not yet supported, you can build the collection locally:
```
Comment (mdellweg) [inline README.md]:
```suggestion
For all versions of Ansible (including < 2.10; where the above approach is not yet supported), you can build the collection locally:
```

Technically i think this is limited to >= 2.8 ...

**Comment by ehelms** [comment]
LGTM after fixing the comment

File: README.md (hunk 1 above)
Comment (wbclark) [reply README.md]: Thanks! With the latest update, I've been more clear and specific with language here, and in a few other spots.

File: README.md [hunk 2]
```diff
@@ -68,6 +68,46 @@ The collection is also available as `ansible-collection-theforeman-foreman` from
 
 After installing the RPM, you can use the modules in the same way as when they are installed directly from Ansible Galaxy.
 
+## Installation From Source
+
+For development or testing purposes, you can install the collection from sources. For production usage, see the instructions above on installing the latest stable release.
```
Comment (ehelms) [inline README.md]:
```suggestion
For development or testing purposes, you can install the collection from source git repository. For production usage, see the instructions above on installing the latest stable release.
```

**Review by ehelms — APPROVED** [review]
Small nit on wording

File: README.md (hunk 2 above)
Comment (wbclark) [reply README.md]: Addressed here, and in the commit message as well.

**Pull request merged**